index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LizanshPortfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+            <div class="header">
+                <img class="img1" src="./img/image_geometry_1.png" alt="">
+                <img class="img2" src="./img/ic_menu.png" alt="">
+            </div>
+    </header>
+
+    <div class="name">
```

Comment:
- [ ] Please update your design to match the original one 

- Original design 👇 

![image](https://user-images.githubusercontent.com/75126481/234728538-d5364808-c181-47bb-b491-dd9344c4de8c.png)

- Your design 👇 

![image](https://user-images.githubusercontent.com/75126481/234728594-bf533d52-2246-42b4-b997-c353fcc2865c.png)